ontotext-yasgui-web-component/src/components/ontotext-yasgui-web-component/ontotext-yasgui-web-component.scss

```diff
@@ -294,6 +298,33 @@
           }
         }
 
+        .show-full-message-link, .show-less-message-link {
+          display: block;
+          font-weight: 400;
+          text-align: right;
+          text-decoration: none;
+        }
+
+        .show-full-message-link::after, .show-less-message-link::after {
+          color: var(--secondary-color);
```

Comment:
also check if this has the correct color in the WB in respect to the new theme colors

Reply:
This is mistake I forgot to remove it. I will add this color in WB MR.

Reply:
Fixed.